engine/release-notes/24.0.md

```diff
@@ -19,6 +19,28 @@ For more information about:
 - Deprecated and removed features, see [Deprecated Engine Features](../deprecated.md).
 - Changes to the Engine API, see [Engine API version history](../api/version-history.md).
 
+## 24.0.1
+
+{% include release-date.html date="2023-05-19" %}
+
+For a full list of pull requests and changes in this release, refer to the relevant GitHub milestones:
+
+- [docker/cli, 24.0.1 milestone](https://github.com/docker/cli/issues?q=is%3Aclosed+milestone%3A24.0.1)
+- [moby/moby, 24.0.1 milestone](https://github.com/moby/moby/issues?q=is%3Aclosed+milestone%3A24.0.1)
+
+### Bug fixes and enhancements
+
+* Fix `image pull` producing an `No such image` error if the `--platform` option was used. [moby/moby#45562](https://github.com/moby/moby/pull/45562)
+* libnetwork: fix a bug where the embedded DNS server would produce an `SERVFAIL` instead of an `NXDOMAIN` error. [moby/moby#45573](https://github.com/moby/moby/pull/45573)
+* containerd-integration: fix images without layers not showing in `docker image ls -a`. [moby/moby#45588](https://github.com/moby/moby/pull/45588)
```

Comment:
Wording is a little confusing for me.
Maybe:

```suggestion
* containerd-integration: fix issue where images which have no layers are omitted from `docker images ls -a`. [moby/moby#45588](https://github.com/moby/moby/pull/45588)

Reply:
Oh! Thought this one was closed automatically; this PR was superseded by https://github.com/docker/docs/pull/17378 (I missed that Bjorn had that one prepared 😅)